recipes/assetid/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "assetid" %}
+{% set version = "0.0.4" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_rev: {{ version }}
+  git_url: "https://github.com/axiom-data-science/{{ name }}"
```

Comment:
Is there any way to get that sha without downloading from Github? The reason I used a tag was because I imagine Github could change this (e.g. compression level) and it may break, unless it is documented somewhere.